.github/workflows/cleanup_images.yml

```diff
@@ -6,13 +6,20 @@ on:
 jobs:
   cleanup:
     name: Clean up
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - name: Check out code.
         uses: actions/checkout@v2
+      - name: Load CI Environemnt from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v2
+        with:
+          env-file: .ci_env
       - name: Clean up outdated images
         env:
           REGISTRY_USER: ${{ secrets.REGISTRY_USER }}
           REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+          DOCKER_ORG: ${{ env.DOCKER_ORGANIZATION }}
         run: |
-          apt-get update && apt-get install jq -y && ./gh-actions-scripts/cleanup_docker_images.sh
+          # sudo apt-get update && sudo apt-get install jq -y
```

Comment:
Is this commented out intentionally?

Reply:
Yes, it looks like those tools are installed automatically on ubuntu 20.04. I'll remove the line.

Reply:
Line removed.